lib/SimplePie/SimplePie/Sanitize.php

```diff
@@ -643,6 +671,17 @@ protected function strip_attr($attrib, $xpath)
 		}
 	}
 
+	protected function rename_attr($attrib, $xpath)
+	{
+		$elements = $xpath->query('//*[@' . $attrib . ']');
```

Comment:
A bit late for my feedback, but some kind of sanitising of `$attrib` could have been good. Not there, but in the public function.

Reply:
please help me to understand. What kind of sanitizing do you mean?

Reply:
If `$attrib` happens to be something like  `]` or any other special character, it will make the XPath crash or worse